File: javascript/exercise1.js
function solution(number){
  let total = 0;
  for (let i=0; i < number; i++) {
    (i % 3 === 0 || i % 5 === 0) ? total += i : null
  }
  return total;
}

function testSolution() {
  if (solution(10) === 23) {
    console.log('Test passed');
  } else {
    console.error('Failed')
  }
}

testSolution();

File: python/exercise1.py
def solution(number):
  total = 0
  for i in range(0, number):
    if i % 3 == 0 or i % 5 == 0:
      total += i
  return total

def testSolution():
  if solution(10) == 23:
    print('Test Passed')
  else:
    print('Failed')

testSolution()

